src/tests/chain.test.ts

```diff
@@ -3,22 +3,33 @@
 // import { toChainEntityName } from '../../../helpers'
 // import { ChainType } from '../../..'
 import { Interfaces, Models, PluginChainFactory, Helpers } from '@open-rights-exchange/chain-js'
-import { ropstenEndpoints } from '../plugin/examples/helpers/networks'
+import nock from 'nock'
+import { startVCR, stopVCR } from '@aikon/network-vcr'
+import { goerliEndpoints as testNetEndpoints } from '../plugin/examples/helpers/networks'
```

Comment:
I don't love the import from the examples folder

Reply:
All the tests import endpoints from there. It doesn't look like they are defined anywhere else. Should they move to a new location to be shared between tests and examples?